bin/modules/validateAddPackages.js

```diff
@@ -7,14 +7,34 @@
 const fs = require("fs");
 const removeAll = require("../utils/removeAll");
 
+const isJsonString = function (str) {
```

Comment:
@Jameskmonger pretty clever. In the future I will move this into it's own `util/` module so that it can be more easily tested.